src/NServiceBus.AcceptanceTesting/Support/IConfigureTestEnvironment.cs

```diff
@@ -0,0 +1,12 @@
+﻿namespace NServiceBus.AcceptanceTesting.Support
+{
+    using System.Collections.Generic;
+    using System.Threading.Tasks;
+
+    public interface IConfigureTestEnvironment
```

Comment:
I know that xml comments are not popular, yet would suggest we put a description on this (what's the purpose of this contract).